libs/docs/src/docs/2.2.0/adding-commands.md

```diff
@@ -32,10 +32,9 @@ related to a specific component.
     commands:
     - id: build
         exec:
-        component: mysql
-        commandLine: mvn clean
-        workingDir: /projects/spring-petclinic
-
+            component: mysql
+            commandLine: mvn clean
+            workingDir: /projects/spring-petclinic
```

Comment:
It would be good to match indentation with other devfile versions of this doc page for consistency.
```suggestion
          component: mysql
          commandLine: mvn clean
          workingDir: /projects/spring-petclinic
```